artefacts/README.md

```diff
@@ -1 +1,16 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Handling of figures
 
+## Table of Contents
+
+* [1 Usage of figures](#1)
+
+<a name="1"></a>
+## 1 Usage of figures
+
+- Figures used in the document should be in Portable Network Graphic (PNG) format
+- Figures must be stored in the `figures` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/`)
+- File name of the figures shold contain the chapter number where the figure is used as a prefix
+- Every figure used in the documents should have an editable version commited to the repository in `ppt(x)` or in `svg` format
+- Editable versions should be stored in the `figures/artefacts` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/artefacts`)
```

Comment:
.svg starts to cross the boundaries between editable artifact and displayable artifact.  Should we place the viewable artifact and the editable artifact in the same directory, in preparation for when the SVG format converges editable/viewable into a single artifact?

Reply:
My worry is that the figures folder becomes overloaded by the editable files. I would add the svg files in the figures folder if they are directly used while keep the editables in the artifacts folder.

Reply:
Ok, so I would interpret your proposal as:
- for SVG files, the only copy is stored in the figures folder, per-chapter, as that is both the editable and the viewable version of the figure
- for PNG files, the viewable form is stored in the figures folder, per-chapter, and the editable copy is stored in the single shared artefacts folder.

Am I understanding properly?

Reply:
@kedmison this is what I understood. if .svg file is used directly in the md then it is placed in figures folder, if it is the editable version of a png file that is used, then it will be placed in the artefacts.